control-plane/pkg/reconciler/channel/v2/channelv2.go

```diff
@@ -576,40 +596,61 @@ func (r Reconciler) reconcileConsumerGroup(ctx context.Context, channel *messagi
 		},
 	}
 
+	expectedCg.Spec.Replicas = ptr.Int32(1) //Must be set for Scaled Object ref
+
+	// TODO: make keda annotation values configurable and maybe unexposed
+	subscriptionAnnotations, err := r.getSubscriptionAnnotations(channel.Namespace, s)
+	if err != nil && !apierrors.IsNotFound(err) {
+		return nil, fmt.Errorf("failed to extract subscription annotations for subscriber %v: %w", s, err)
+	}
+	expectedCg.Annotations = kedafunc.SetAutoscalingAnnotations(subscriptionAnnotations)
+
+	if secret != nil {
+		secretName = secret.Name
+		//secretNamespace := secret.Namespace
```

Comment:
Can we remove this commented line?

Reply:
Yes, its not there anymore in the latest code..